draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -413,3 +413,8 @@ This could be described in {{ICESDP}} and/or {{WebRTCSpec}}.
 
 The proposal allows updating {{IPHandling}} so that mode 2 is not the mode used by default when user consent is not required.
 Instead, the default mode could be defined as mode 3 with mDNS-based ICE candidates.
+
+IANA Considerations
```

Comment:
Let's put this right after Security Considerations. I think the Spec Requirements section will more elsewhere in a future rev.

Reply:
Done